app/views/merchant/items/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<center><h1>My Items</h1> </center>
+
+<div class="container">
+  <h4>Item Information</h4>
+    <table class="table">
+      <thead class="thead-light">
+        <tr>
+          <th scope="col">Item Name</th>
+          <th scope="col">Item ID</th>
+          <th scope="col">Image</th>
+          <th scope="col">Description</th>
+          <th scope="col">Price</th>
+          <th scope="col">Quantity</th>
+          <th scope="col">Active?</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% @items.each do |item| %>
+          <section id='item-<%= item.id %>'>
+              <tr>
+                <td><%= item.name %> </td>
+                  <td><%= item.id %></td>
+                <td><img id='thumbnail-<%= item.id %>'><%= image_tag item.image, class: 'img_thumbnail' %></img></td>
+                <td><%= item.description %></td>
+                <td><%= item.price %></td>
+                <td><%= item.inventory %></td>
+                <td><%= item.active? %></td>
+              </tr>
+          </section>
+        <% end %>
+      </tbody>
+    </table>
+</div>
```

Comment:
Nice use of the bootstrap table layout!

Reply:
This looks great! Maybe in future user stories, `Item ID` might need to be clickable